CONTRIBUTING.md

```diff
@@ -57,7 +59,27 @@ early and often.
 
 We are using [conventional
 commits](https://www.conventionalcommits.org/en/v1.0.0/) so please format your
-commit messages accordingly.
+commit messages accordingly. There is a github action that uses the
+[gitlint](https://jorisroovers.com/gitlint/latest/) linter to check your commit
+messages for validity.
+
+Please try to use a scope when writing a commit message. APIS consists of
+various individual modules, so the scope tells the users which module your
+change refers to.
+
+We are using the [release-please](https://github.com/googleapis/release-please)
+github action, which uses the information from the conventional commit messages
+to generate changelogs and bump the version. Therefore it is important to use
+the correct commit type in the commit message, i.e. `fix` for smaller fixes,
+`feat` for new features. Breaking changes have to have an exclamation mark
+(`!`) after the type/scope, but please also explain the breaking change in the
+commit message body.
+
+
+## Workflows
+
+You can see a list of worklows that check pull requests
+[here](https://github.com/acdh-oeaw/apis-core-rdf/tree/main/.github/workflows).
```

Comment:
Would change to "workflows which are run on pull requests" as checking implies inspection for mistakes, rule-following etc., which is what the earlier referenced linting & code formatting actions do, but which doesn't apply to _all_ workflows.